components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/basic-policy-update.jsp

```diff
@@ -284,14 +285,14 @@
     }
 
     if(completedRule != null && completedRule.equals("true")){
-        forwardTo = nextPage + ".jsp?";
+        forwardTo = Encode.forUriComponent(nextPage) + ".jsp?";
```

Comment:
Do not encode this one. Encode only untrusted added to the query string with uri component